lib/matplotlib/tests/test_contour.py

```diff
@@ -309,3 +309,53 @@ def test_contourf_symmetric_locator():
     locator = plt.MaxNLocator(nbins=4, symmetric=True)
     cs = plt.contourf(z, locator=locator)
     assert_array_almost_equal(cs.levels, np.linspace(-12, 12, 5))
+
+
+def test_contour_1x1_array():
+    # github issue 8197
+    try:
+        plt.contour([[0]])
+    except TypeError as exc:
+        assert exc.args[0] == 'Input z must be at least a 2x2 array.'
```

Comment:
we have recently changed over to using py.test. You can see how asserts are tested here: http://doc.pytest.org/en/latest/assert.html#assertions-about-expected-exceptions